curator-recipes/src/main/java/org/apache/curator/framework/recipes/cache/PathChildrenCache.java

```diff
@@ -184,6 +184,28 @@ public PathChildrenCache(CuratorFramework client, String path, boolean cacheData
     }
 
     /**
+     * @param client        the client
```

Comment:
I was trying to avoid using the ctor's with the `dataIsCompressed` argument since `ServiceCacheImpl` did not use those ctor's. I suppose it is fine to just invoke those ctor's and pass in `false` for `dataIsCompressed`.